gravitee-apim-console-webui/src/management/api/portal/user-group-access/groups/api-portal-access-groups.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form *ngIf="form" [formGroup]="form" class="api-portal-access-groups">
+  <mat-card>
+    <div class="api-portal-access-groups__body">
+      <mat-form-field appearance="fill" *ngIf="!isReadOnly; else readOnlyGroups">
```

Comment:
Did you ask Audrey if she would approve appearance="outline" (default value) ? I feel we should try to unify our form fields whenever it's possible

Reply:
Checked in with her for fill vs outline --> outline to be used.